test/utils/node/cache.ts

```diff
@@ -0,0 +1,127 @@
+import { expect } from 'chai';
+import {
+	isCacheRunning,
+	startCache,
+	stopCache,
+} from '../../../src/utils/node/cache';
+import { NETWORK } from '../../../src/utils/constants';
+import * as workerProcess from '../../../src/utils/worker-process';
+import * as nodeConfig from '../../../src/utils/node/config';
+
+describe('cache node utils', () => {
+	describe('#isCacheRunning', () => {
+		let workerProcessStub: any = null;
+		beforeEach(() => {
+			workerProcessStub = sandbox.stub(workerProcess, 'exec');
+		});
+
+		describe('when installation does not exists', () => {
```

Comment:
instead of `installation`, maybe like `when redis server is not installed`?
same for below